frontend/src/Startups/startup.css

```diff
@@ -0,0 +1,104 @@
+.section-header h3 {
+  font-size: 36px;
+  color: #fff;
+  text-align: center;
+  font-weight: 500;
+  position: relative;
+}
+
+.section-header p {
+  text-align: center;
+  margin: auto;
+  font-size: 15px;
+  padding-bottom: 60px;
+  color: #fff;
+  width: 50%;
+  
+}
+
+#start {
+  background:url(../Images/start/wp.jpg);

```

Comment:
@naveennvrgup sure won't break?

Reply:
@kdpisda I had a video call with frontend guys to not do this immediately after this commit. explained them how to fetch this from our apis and do this dyanamically the react way.